test/case/01_bootstrap_update

```diff
@@ -18,4 +18,16 @@ cd $repo
 export REPO_NAME=${0##*/}
 
 make update_boilerplate
+
+check_update 
+
+if [ $? -ne 0 ] ; then
+	exit $?
+fi
+
+add_convention test/test_base_convention
+make update_boilerplate
+
+check_update
```

Comment:
I tried to get this to fail by adding a line here that modifies one of the convention files before `check_update`. It didn't work. I didn't dig in.